### DISCUSSION TITLE
Is flexbox or grid works in cssBattles?

### DISCUSSION THREAD

**Discussion by hiharish-sh** [cssbattle/forum#42] [Q&A]
?

**Comment by SelenIT** [comment]
They both do work. But usually they require longer code comparing to top solutions that don't redefine `display`.

**Reply by gunnarbittersmann** [reply]
I’ve never used grid yet; `grid-template` would need too many characters, I guess.

I’ve used `display:flex` and `flex:1` on some targets – later replaced by shorter solutions in most cases. But it’s still in my shortest solution to \#37 Tunnel. Maybe that’s why my shortest solution is sooo long. 🤨

**Reply by SelenIT** [reply]
I had some hope that something like `grid:53q 53q/repeat(4,53q);gap:5vw;` would help to arrange items as needed in [#28 Cups & Balls](https://cssbattle.dev/play/28). It actually worked, but the solution ended up 26 chars longer than a simpler `float`-based one. Regarding [#37](https://cssbattle.dev/play/37), `margin`s and/or `padding`s worked well for me without needing to change `display` for anything 🤔
